src/storage-ors-preview/azext_storage_ors_preview/_params.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+from knack.arguments import CLIArgumentType
+from ._validators import get_datetime_type
+
+
+def load_arguments(self, _):
+
+    from azure.cli.core.commands.parameters import get_resource_name_completion_list
+
+    acct_name_type = CLIArgumentType(options_list=['--account-name', '-n'], help='The storage account name.',
+                                     id_part='name',
+                                     completer=get_resource_name_completion_list('Microsoft.Storage/storageAccounts'))
+    ors_policy_type = CLIArgumentType(
+        options_list=['--policy', '-p'],
+        help='The object replication policy definition between two storage accounts, in JSON format. '
+             'Multiple rules can be defined in one policy.'
+    )
+    policy_id_type = CLIArgumentType(
+        options_list=['--policy-id'],
+        help='The ID of object replication policy or "default" if the policy ID is unknown.'
+    )
+    rule_id_type = CLIArgumentType(
+        options_list=['--rule-id', '-r'],
+        help='Rule Id is auto-generated for each new rule on destination account. It is required '
+             'for put policy on source account.'
+    )
+    prefix_math_type = CLIArgumentType(
+        nargs='+', arg_group='Filters',
+        help='Optional. Filter the results to replicate only blobs whose names begin with the specified '
+             'prefix.'
+    )
+    min_creation_time_type = CLIArgumentType(
+        options_list=['--min-creation-time', '-t'], arg_group='Filters', type=get_datetime_type(True),
+        help="Blobs created after the time will be replicated to the destination. It must be in datetime format "
+             "'yyyy'-'MM'-'dd'T'HH':'mm':'ss'Z'. Example: 2020-02-19T16:05:00Z")
```

Comment:
yyyy-MM-dd HH:mm:ss? Make more sense to me without `'`

Reply:
sure.